deploy/bootstrap/istio/03-istio-ingress.yaml

```diff
@@ -5,15 +5,7 @@ metadata:
   name: istio-default
 spec:
   profile: default
-  # PRODUCTION READINESS CHANGE REQUIRED
-  # The Istio images should be sourced from a non-public container registry, such as the
-  # one deployed along side of this reference implementation.
-  # - az acr import --source docker.io/istio/operator:1.11.3 -n <your-acr-instance-name>
-  # - az acr import --source docker.io/istio/pilot:1.11.3 -n <your-acr-instance-name>
-  # - az acr import --source docker.io/istio/proxyv2:1.11.3 -n <your-acr-instance-name>
-  # and then set this to
-  # hub: <your-acr-instance-name>.azurecr.io
-  hub: docker.io/istio
+  hub: acraks6jqf6qyt7jarg.azurecr.io/istio
```

Comment:
Should we use ghcr instead?  I think we opened access from the cluster to it and our intent was to move there.

Reply:
@sivamu the current setup only work for dev cluster, since each cluster has a unique ACR instance. This file is inside the bootstrap folder, thus it will be picked up by the preprod cluser as well. We have to discuss using ghcr

Reply:
Adding them to GHCR for now. will discuss later how to handle external images for a long term solution